completions/bash/skopeo

```diff
@@ -77,6 +77,7 @@ _skopeo_inspect() {
      --cert-dir
      --retry-times
      --registry-token
+     --format
```

Comment:
nit
could you move this up a few lines to alphabetize?

Reply:
Fixed

Reply:
You said fixed, but this review isn't reflecting that.